ignition/examples.md

```diff
@@ -40,10 +40,12 @@ WantedBy=multi-user.target
 
 In many scenarios, it may be preferable to use btrfs for the root filesystem.
 This config will find the device with the filesystem label "ROOT" (the root
```

Comment:
"This example Ignition configuration will locate the device with the "ROOT" filesystem label..."